libs/blocks/comparison-table/comparison-table.js

```diff
@@ -0,0 +1,511 @@
+import { createTag, getConfig } from '../../utils/utils.js';
+import { getMetadata } from '../section-metadata/section-metadata.js';
```

Comment:
Feel free to insta-resolve this, but: this could be a bit misleading, since metadata usually refers to page metadata, while this refers to section metadata, it might not be a bad idea to import this under `getSectionMetadata`. This way, if in the future we need to base logic on actual page metadata, we can use the utils method easily.